1-org/modules/centralized-logging/variables.tf

```diff
@@ -34,6 +34,18 @@ variable "resource_type" {
   }
 }
 
+variable "billing_account" {
+  description = "Billing Account ID used in case sinks are under billing account level. Format 000000-000000-000000."
+  type        = string
+  default     = null
+}
+
+variable "enable_billing_account_sink" {
+  description = "If true, a log router sink will be created for the billing account. The billing_account variable cannot be null."
+  type        = bool
+  default     = true
```

Comment:
I think the default should be `false` because the billing account default value is null, which would make the module if the user did not change one of the values.

changing the default to `false` will also require that the call to the module uses 

```
billing_account                = local.billing_account
enable_billing_account_sink    = true
```